webapp/app.py

```diff
@@ -637,6 +652,250 @@ def takeovers_index():
 app.add_url_rule("/", view_func=template_finder_view)
 app.add_url_rule("/<path:subpath>", view_func=template_finder_view)
 
+# Data Platform Spark on K8s docs
+data_spark_k8s_docs = Docs(
+    parser=DocParser(
+        api=charmhub_discourse_api,
+        index_topic_id=8963,
+        url_prefix="/data/docs/spark/k8s",
+    ),
+    document_template="/data/docs/spark-k8s/document.html",
+    url_prefix="/data/docs/spark/k8s",
+    blueprint_name="data-docs-spark-k8s",
+)
+app.add_url_rule(
+    "/data/docs/spark/k8s/search",
+    "data-docs-spark-k8s-search",
+    build_search_view(
+        session=session,
+        site="ubuntu.com/data/docs/spark/k8s",
+        template_path="/data/docs/spark-k8s/search-results.html",
+        search_engine_id=search_engine_id,
+    ),
+)
+data_spark_k8s_docs.init_app(app)
+
+# Data Platform MySQL on IAAS docs
+data_mysql_iaas_docs = Docs(
+    parser=DocParser(
+        api=charmhub_discourse_api,
+        index_topic_id=9922,
```

Comment:
The correct id is [9925](https://discourse.charmhub.io/t/charmed-mysql-documentation/9925).

Reply:
Thanks, that is better :+1: